lab_3_generate_by_ngrams/main.py

```diff
@@ -104,6 +155,11 @@ def _put(self, element: str) -> None:
         In case of corrupt input arguments or invalid argument length,
         an element is not added to storage
         """
+        if not isinstance(element, str) or len(element) != 1:
+            return None
+        if element not in self._storage:
+            self._storage[element] = len(self._storage)
+        return self._storage[element]
```

Comment:
According to the type annotation, it should always return `None`

Reply:
Fixed